### ISSUE TITLE
Todo List 

### ISSUE THREAD

**Issue by SumitKanth** [Design-and-Code/Quick-projects#7]
I want to add **Todo List** project.
Tech used is **React**.
Can you plz assign me ?

**Comment by SumitKanth** [comment]
@Greeshma2903 thanks for assigning me, I have 2 doubts !
1.  Is my contribution will be added on this hacktoberfest event ?
2.  We have to give link of github project repo and not to add files n?
ex -  `sh add-projects.sh vanilla https://github.com/Greeshma2903/links-card`

**Comment by Greeshma2903** [comment]
> @Greeshma2903 thanks for assigning me, I have 2 doubts !
> 
> 1. Is my contribution will be added on this hacktoberfest event ?
> 2. We have to give link of github project repo and not to add files n?
>    ex -  `sh add-projects.sh vanilla https://github.com/Greeshma2903/links-card`

1. Yes
2. Yes, just running the script on your terminal or Git Bash should do the work (basically it adds the project as submodule)

**Comment by SumitKanth** [comment]
@Greeshma2903 The command given in the  **Contribution giudelines** are not working 
### Showing this !
```chmod +x add-projects.sh
chmod: cannot access 'add-projects.sh': No such file or directory
```

**Comment by Greeshma2903** [comment]
> @Greeshma2903 The command given in the **Contribution giudelines** are not working
> 
> ### Showing this !
> ```
> chmod: cannot access 'add-projects.sh': No such file or directory
> ```

Ohh, this shouldn't ideally happen. Can you send an SS? 
Try running it in the root folder, and not in the subfolders 🤔

**Comment by SumitKanth** [comment]
@Greeshma2903 Which SS ?
I am following same procedure given in **Contribution Guidelines**

**Comment by Greeshma2903** [comment]
> @Greeshma2903 Which SS ? I am following same procedure given in **Contribution Guidelines**

Sorry, I meant a screenshot of the issue.

**Comment by SumitKanth** [comment]
@Greeshma2903 

![Screenshot (565)](https://user-images.githubusercontent.com/90907720/194718186-f340f933-a2e1-4ac8-8f4d-596021d82edb.png)

**Comment by Yashkapure06** [comment]
> @Greeshma2903
> 
> ![Screenshot (565)](https://user-images.githubusercontent.com/90907720/194718186-f340f933-a2e1-4ac8-8f4d-596021d82edb.png)

Don't worry 

Just go ahead and run next command 
``` sh add-project.sh <folder_name> <github_project_url> ```

**Comment by Greeshma2903** [comment]
Ahh yes, you can run the next cmd as Yash said.😅

**Comment by SumitKanth** [comment]
@Yashkapure06 @Greeshma2903 okii, thanks👀

**Comment by SumitKanth** [comment]
@Yashkapure06 @Greeshma2903 
`sh add-project.sh <folder_name> <github_project_url>` used this command then also showing `sh: add-projects.sh: No such file or directory`

**Comment by Greeshma2903** [comment]
> @Yashkapure06 @Greeshma2903 `sh add-project.sh <folder_name> <github_project_url>` used this command then also showing `sh: add-projects.sh: No such file or directory`

Sumit, do you mind joining our community and ping me there, since the issue is still persisting? 
Join Link: https://discord.com/invite/gM3bG4rAU5 (Discord)

**Comment by Greeshma2903** [comment]
> @Yashkapure06 @Greeshma2903 `sh add-project.sh <folder_name> <github_project_url>` used this command then also showing `sh: add-projects.sh: No such file or directory`

So, in your cmd, I realised the name of the file should be `add-project.sh` (without an extra 's' at the end)

**Comment by Yashkapure06** [comment]
Yeah! it is ```add-project.sh``` for anyone who forks this repo. I think we will have to make changes in main add-prjects.sh -> add-prject.sh

**Comment by SumitKanth** [comment]
> > @Yashkapure06 @Greeshma2903 `sh add-project.sh <folder_name> <github_project_url>` used this command then also showing `sh: add-projects.sh: No such file or directory`
> 
> Sumit, do you mind joining our community and ping me there, since the issue is still persisting? Join Link: https://discord.com/invite/gM3bG4rAU5 (Discord)

I have already joined your discord
